Global/README.md

```diff
@@ -81,3 +81,7 @@ E.g. changes "Do androids dream of electric sheep?" to "Do Androids Dream of Ele
 
 Toggles between upper- and lower-case letters in selected text.
 
+### [Run your's preferred compare tools based on the last two items on clipboard history](do-compare-based-on-last-two-clipboard-items.ini)
```

Comment:
The title should be something short here so it's easier to refer to it later with a shorter more permanent link. E.g.: "Compare Last Two Clipboard History Items"